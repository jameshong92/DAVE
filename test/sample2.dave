// test operations
int main() {
	str s1 = "hello ";
	str s2 = "world!";
	str s3 = s1 + s2;
	str s4 = "dave " + "is " + "completed!";
    print(s3);
    print(s4);
}