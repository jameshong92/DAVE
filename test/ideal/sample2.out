hello world!
dave is completed!
